### PR TITLE
update mainnet token list

### PR DESCRIPTION
**Pull request by p6te**
- update mainnet token list

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -367,6 +365,14 @@ export const getTokenDataByAddresses = async (
   psp22: PSP22,
   address: string
 ): Promise<Record<string, Token>> => {
+  const knownTokens: Record<string, Token> = {}
```
Comment (zielvna) [inline src/utils/utils.ts:368]: i don't think we should prevent getting known tokens data inside this function and we should prevent passing known tokens instead in other parts of the app

File: src/utils/utils.ts [hunk 2]
```diff
@@ -993,6 +997,7 @@ export const getNewTokenOrThrow = async (
 ): Promise<Record<string, Token>> => {
   const tokenData = await getTokenDataByAddresses([address], psp22, walletAddress)
 
+  console.log('tokenData', tokenData)
```
Comment (zielvna) [inline src/utils/utils.ts:1000]: leftover log

**Review by zielvna — CHANGES_REQUESTED** [review]

File: src/utils/utils.ts (hunk 1 above)
Comment (p6te) [reply src/utils/utils.ts:368]: for this one, i had the problem that two tokens from URL were loaded as unknown tokens, but they were I mainnet token list.

File: src/store/consts/static.ts [hunk 3]
```diff
@@ -18,6 +18,25 @@ import {
 } from './types'
 import { bestTiersCreator } from '@utils/utils'
 import { POSITIONS_ENTRIES_LIMIT } from '@invariant-labs/a0-sdk/target/consts'
+import mainnetListJson from '@store/consts/tokenLists/mainnet.json'
+
+export const WAZERO_ADDRESS = {
+  ...WAZERO_ADDRESS_SDK,
+  [Network.Mainnet]: '5CtuFVgEUz13SFPVY6s2cZrnLDEkxQXc19aXrNARwEBeCXgg'
```
[the remark this answers is not in the document]
Comment (p6te) [reply src/store/consts/static.ts:25]: i took that addresses from mainnet token list. so we don't duplicate that common tokens. and that logic is on frontend so I overwritten them. Or we should keep both tokens? like our BTC and wBTC from token list?

File: src/fetchMainnetList.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import fs from 'fs'
+import axios from 'axios'
+
+const run = async () => {
+  const tokensObject = (await axios.get('https://api.psplist.xyz/tokens')).data
```
Comment (wojciech-cichocki) [inline src/fetchMainnetList.js:5]: How long is this call taking (maybe it's worth caching it on the frontend)?

File: src/store/consts/static.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (wojciech-cichocki) [reply src/store/consts/static.ts:25]: We can remove tokens that were used for the mainnet test. The tokens from mainnet are sufficient

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Add also USDT to common tokens:
image

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**